.travis.yml

```diff
@@ -19,7 +22,9 @@ before_install:
   - GALAXY_USER_PASSWD=admin
   - BIOBLEND_GALAXY_API_KEY=admin
   - BIOBLEND_GALAXY_URL=http://localhost:8080
+  - TEST_TOOLSET_URL=https://raw.githubusercontent.com/afgane/docker-galaxy-stable/dev/galaxy/test_toolset.yml
```

Comment:
I guess you're right - this started out as an object in S3 while testing so just didn't think about it. I'll update it.